release_notes/45-release-notes.adoc

```diff
@@ -0,0 +1,416 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="release-notes-45"]
+= Red Hat Advanced Cluster Security for Kubernetes 4.5
+include::modules/common-attributes.adoc[]
+:context: release-notes-45
+
+toc::[]
+
+{rh-rhacs-first} is an enterprise-ready, Kubernetes-native container security solution that protects your vital applications across the build, deploy, and runtime stages of the application lifecycle.
+{product-title} deploys into your infrastructure and integrates with your DevOps tools and workflows. This integration provides better security and compliance, enabling DevOps and InfoSec teams to operationalize security.
+
+.Release dates
+[options="header"]
+|====
+
+|{product-title-short} version |Released on
+
+|`4.5.0` | 18 July 2024
+
+|====
+
+[id="about-this-release-450_{context}"]
+== About this release
+
+{product-title-short} 4.5 includes the following new features, improvements, and updates:
+
+Compliance::
+* TBD: links to new features
+
+Network::
+* TBD: links to new features
+
+Platform::
+* TBD: links to new features
+
+Policy::
+* TBD: links to new features
+
+Vulnerability Management::
+* TBD: links to new features
+
+[id="new-features_{context}"]
+== New features
+
+This release adds improvements related to the following components and concepts:
+
+[id="vulnerability-management-v2-ga_{context}"]
+=== Vulnerability Management 2.0 is generally available
+
+With this release, Vulnerability Management 2.0 is generally available. It provides an enhanced user experience with intuitive navigation.
+
+* It includes persona-specific vulnerability management views, Workload CVEs, Platform CVEs, and Node CVEs.
+* It provides actionable data to triage and remediate vulnerabilities efficiently.
+* Its default filter view now persists across user sessions.
+* You can download comprehensive vulnerability on-demand reports scoped with collections.
+* It includes enhanced Exception management workflows with audit capabilities.
+
+[NOTE]
+====
+As part of this update, the *Risk Acceptance* workflow is replaced with *Exception Management*. When you upgrade to {product-title-short} 4.5:
+
+* Existing deferrals and false positive requests will be migrated to *Exception Management*.
+* Globally snoozed Image CVEs will be migrated to create approved deferrals under *Exception Management*.
+====
+
+
+[id="compliance-20-updates_{context}"]
+=== Compliance 2.0 updates
```

Comment:
Compliance 2.0 is obsolete as of 4.5. I think the same applies to vulnerability management.